### DISCUSSION TITLE
Is it possible to add members to a guild?

### DISCUSSION THREAD

**Discussion by bvtrfly** [discord-php/DiscordPHP#1230] [Q&A] [answered]
The discord API makes this possible by providing the users access token:
[https://discord.com/developers/docs/resources/guild#add-guild-member](https://discord.com/developers/docs/resources/guild#add-guild-member)

Is there a way to achieve this using discordPHP?

**Comment by valzargaming** [comment, marked as answer]
This requires being in the scope of [oauth2](https://discord.com/developers/docs/topics/oauth2), so no, not out of the box. You would want to look at using something like [MarkisDev/discordoauth](https://github.com/MarkisDev/discordoauth/blob/master/discord.php) for that kind of thing.

I wrote a [class](https://github.com/Valgorithms/Civilizationbot/blob/cc4842b0a4ac358af3513b5b1a9df42685b7dd8b/src/Civ13/DiscordWebAuth.php#L15) that lets me do something similar in one of my bots, but again this is in the context of ReactPHP HttpServer.

**Reply by bvtrfly** [reply]
Thanks for the help! I will look into it 😄
